packages/qgrid-core/src/highlight/highlight.let.js

```diff
@@ -78,6 +77,36 @@ export class HighlightLet {
 			}
 		});
 
+		this.oneRow = new Command({
+			source: 'highlight.oneRow',
+			canExecute: () => !this.isRendering,
+			execute: (row, state) => {
```

Comment:
it should not be a state argument, this command just should highlight a row, not to `unhighlight`